distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -621,7 +621,7 @@ type Date
 
              example_add = Date.new 2020 - Date_Period.Month
     - : Period | Date_Period -> Date ! (Time_Error | Illegal_Argument)
-    - self amount:(Period|Date_Period) =
+    - self amount:(Period | Date_Period) =
```

Comment:
Move `amount:` inside the parentheses? I notice you did all three date +/- pairs like this, is there a reason?

Reply:
made my simple checker match the types on the two line. 
Eventually we should agree a style guide for the signature.